internal/bind/wide_time_types.go

```diff
@@ -12,6 +13,23 @@ func (m WideTimeTypes) ToYdb(sql string, args ...any) (yql string, newArgs []any
 	newArgs = make([]any, 0, len(args))
```

Comment:
I think about `WideTimeTypes` whould be function with bool argument (for allow to change default in future without additional option).